OutaTime/TimeCircutsViewController.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  TimeCircutsViewController.swift
+//  OutaTime
+//
+//  Created by Lambda_School_Losaner_256 on 1/29/20.
+//  Copyright © 2020 Benglobal Creative. All rights reserved.
+//
+
+import UIKit
+
+class TimeCircutViewController: UIViewController {
+    
+    var timer: Timer? = nil
+    var currentSpeed: Int = 0
+    
+    var dateFormatter: DateFormatter {
+        
+        let formatter = DateFormatter()
+        formatter.dateFormat = "MMM dd yyyy"
+        formatter.timeZone = TimeZone(secondsFromGMT: 0)
+        return formatter
+    }
+    
+    @IBOutlet weak var destinationTimeLabel: UILabel!
+    @IBOutlet weak var presentTimeLabel: UILabel!
+    @IBOutlet weak var lastTimeDepartedLabel: UILabel!
+    @IBOutlet weak var speedLabel: UILabel!
+    
+    @IBAction func travelBackTapped(_ sender: UIButton) {
+        timer = Timer.scheduledTimer(withTimeInterval: 0.1, repeats: true, block: updateSpeed(timer:))
+    }
+    
+    
+    func resetTimer(){
+        timer?.invalidate()
+        timer = nil
+        
+        
+        
+        var destinationDate: Date?
+        
+        
+        
+        let now = Date()
+        
+        
+        
+    }
```

Comment:
This closing brace needs to be on line 37.
It is including the variable on line 40 and the constant on line 44 in the scope of the resetTimer function.